cmd/flags_node.go

```diff
@@ -9,34 +9,56 @@ import (
 	flag "github.com/spf13/pflag"
 
 	"github.com/celestiaorg/celestia-node/node"
+	"github.com/celestiaorg/celestia-node/params"
 )
 
 var (
-	nodeStoreFlag  = "node.store"
-	nodeConfigFlag = "node.config"
+	nodeStoreFlag   = "node.store"
+	nodeConfigFlag  = "node.config"
+	nodeNetworkFlag = "node.network"
 )
 
 // NodeFlags gives a set of hardcoded Node package flags.
-func NodeFlags(tp node.Type) *flag.FlagSet {
+func NodeFlags() *flag.FlagSet {
 	flags := &flag.FlagSet{}
 
 	flags.String(
 		nodeStoreFlag,
-		fmt.Sprintf("~/.celestia-%s", strings.ToLower(tp.String())),
+		"",
 		"The path to root/home directory of your Celestia Node Store",
 	)
 	flags.String(
 		nodeConfigFlag,
 		"",
 		"Path to a customized node config TOML file",
 	)
+	flags.String(
+		nodeNetworkFlag,
+		"",
+		"The name of the network to connect to",
```

Comment:
nit: we can list all the networks from params pkg here.

Reply:
This would be nice to include `e.g. arabica or mamaki`